controller/src/main/java/com/softserve/borda/controllers/BoardController.java

```diff
@@ -269,14 +269,14 @@ public Board getBoard(@PathVariable Long id) {
                                                                      @PathVariable Long ticketId,
                                                                      @PathVariable String boardId) {
         try {
-            BoardList oldBoardList = boardListService.getBoardListById(oldBoardListId);
-            BoardList newBoardList = boardListService.getBoardListById(newBoardListId);
+            BoardColumn oldBoardColumn = boardColumnService.getBoardListById(oldBoardListId);
+            BoardColumn newBoardColumn = boardColumnService.getBoardListById(newBoardListId);
             Ticket ticket = ticketService.getTicketById(ticketId);
-            oldBoardList.getTickets().remove(ticket);
-            newBoardList.getTickets().add(ticket);
-            boardListService.update(oldBoardList);
-            newBoardList = boardListService.update(newBoardList);
-            BoardListDTO boardListDTO = modelMapper.map(newBoardList, BoardListDTO.class);
+            oldBoardColumn.getTickets().remove(ticket);
```

Comment:
Better to use JPQL and just update boardId on already existing ticket.

Reply:
Got it, will look into it.